packages/types/v1/webhooks.ts

```diff
@@ -7,6 +7,7 @@ export const ZWebhook = z.object({
   createdAt: z.date(),
   updatedAt: z.date(),
   url: z.string().url(),
+  source: z.enum(["user", "zapier"]).nullish(),
```

Comment:
I think it can not be `undefined` or `null` as we have a default value of `"user"` for all existing and new entries.